src/__tests__/theme-provider.js

```diff
@@ -109,3 +110,69 @@ test('does nothing when receive same theme via props', () => {
   wrapper.setProps({theme})
   expect(wrapper).toMatchSnapshotWithGlamor(`with theme prop of margin 2px`)
 })
+
+test('does not override user defined theme contexts', () => {
+  const Child = glamorous.div((props, context) => ({
+    fontSize: context.theme.fontSize,
+  }))
+  Child.contextTypes = {
+    theme: PropTypes.object,
+  }
+
+  class Parent extends React.Component {
```

Comment:
Ah - now it wants `getChildContextTypes` 😅 

Bit of a rabbit hold - I'll keep the `ThemeProvider` ones as-is, but the test in `index.js` works great with the `enzyme` `context` injection.